### ISSUE TITLE
youtube Embed not working properly (not displaying) with some url format

### ISSUE THREAD

**Issue by mynameisek** [adcpm/busy#324]
https://busy.org/drones/@claudiop63/leisure-drones-most-popular-drone-use-case-by-now
![image](https://cloud.githubusercontent.com/assets/8390894/22620783/731d8aee-eb13-11e6-951b-d8610fae6491.png)

video url format : https://youtu.be/CofuD406kyI
Working properly on Steemit: https://steemit.com/drones/@claudiop63/leisure-drones-most-popular-drone-use-case-by-now

**Comment by alexpmorris** [comment]
actually, I think this is a new issue because I believe it was fine last I was on busy.org a week or so ago.  It also seems to effect images as well, not just youtube (any youtube link at that).  The post summaries are fine (even for the drone post).  The problem lies in the full post view, where it seems the video/image rendering phase is being skipped over (probably due to some missed exception handling).

**Comment by nil1511** [comment]
Yes. It use to work earlier because embed were directly applied before processing the body. That resulted in unexpected behaviour like https://github.com/adcpm/busy/issues/309. Then the logic for embeds was changed to apply after processing the input and only apply on links.

Remarkable should linkify them to links. As of now I have added to process embed url which are not in any tag or in anchor tag.
